recipe/bld.bat

```diff
@@ -0,0 +1,5 @@
+set TCL_LIB_PATH=%PREFIX%\Library\lib\tcl8.6\%PKG_NAME%%PKG_VERSION%
+mkdir %TCL_LIB_PATH%
```

Comment:
Please add a safety net checking if the directory exists. Or add a line raising errors: `if errorlevel 1 exit 1` see here https://github.com/conda-forge/r-base-feedstock/blob/19c643d97f43050424be0686a4d22dd88fcc5e36/recipe/bld-r-base.bat#L18

the following lines could also need this precaution.

Reply:
Thanks for the hint @fhoehle. I will add the error-checking commands. 
I didn't plan to switch to bwidget build in r-base.

Reply:
Thanks adding